src/components/Common/OrganizationCard.js

```diff
@@ -84,15 +84,16 @@ class OrganizationCard extends Component {
             name, categoryautosortscript, overview, location, website, facebookUrl,
             instagramUrl, twitterUrl, phone
         } = this.props.organization;
-        let distance, distanceElement;
+        let distance, distanceElement, distanceURL;
         let url = this.validatedUrl(website);
-        if (this.props.currentPos) {
+        if (this.props.currentPos && this.props.organization.coordinates) {
             distance = getDistance({coordinates: this.props.organization.coordinates}, this.props.currentPos);
```

Comment:
I would add coordinates to the list of const at the start of the render method so this way you dont have to keep retyping "this.props.organization", it will look cleaner and more consistence with the rest of the destructured variables.